docs/user/dashboard/lens-end-to-end/lens-end-to-end.asciidoc

```diff
@@ -0,0 +1,506 @@
+[float]
+[[lens-end-to-end]]
+== Try it: Analyze website data on a dashboard
+
+You have collected data about your webiste, and you're ready to find the answers to some key questions. 
```

Comment:
I think a newbie user would be confused if this isn't their use case, so we should explain more:

```suggestion
Most dashboards use similar types of analysis to answer key questions. In this tutorial you will base your analysis off of {kib} sample data which resembles logs from the Apache web server.
```

Reply:
I like Wylie's suggestion too, there's also a small typo in the line 5: `webiste`